tests/test_all_rules.py

```diff
@@ -133,22 +133,15 @@ def build_rule(query, bbr_type="default", from_field="now-120m", interval="60m")
 
     def test_max_signals_note(self):
         """Ensure the max_signals note is present when max_signals > 1000."""
-        max_signal_standard_setup = 'This rule is configured to generate more **Max alerts per run** than the ' \
-                                    'default 1000 alerts per run set for all rules. This is to ensure that it ' \
-                                    "captures as many alerts as possible.\n\n**IMPORTANT:** The rule's " \
-                                    '**Max alerts per run** setting can be superseded by the ' \
-                                    '`xpack.alerting.rules.run.alerts.max` Kibana config setting, which determines ' \
-                                    'the maximum alerts generated by _any_ rule in the Kibana alerting framework. ' \
-                                    'For example, if `xpack.alerting.rules.run.alerts.max` is set to 1000, this rule ' \
-                                    'will still generate no more than 1000 alerts even if its own **Max alerts per ' \
-                                    'run** is set higher.\n\nTo make sure this rule can generate as many alerts as ' \
-                                    "it's configured in its own **Max alerts per run** setting, increase the " \
-                                    '`xpack.alerting.rules.run.alerts.max` system setting accordingly.\n\n**NOTE:** ' \
-                                    'Changing `xpack.alerting.rules.run.alerts.max` is not possible in Serverless ' \
-                                    'projects.'
+        max_signal_standard_setup = 'For information about troubleshooting maximum alerts warning '\
+                                    'please refer to this [guide]'\
+                                    '(https://www.elastic.co/guide/en/security/current/alerts-ui-monitor.html#troubleshoot-max-alerts).'  # noqa: E501
         for rule in self.all_rules:
             if rule.contents.data.max_signals and rule.contents.data.max_signals > 1000:
```

Comment:
We should still be using .get here right? Prior comment was a reference to comparing `rule.contents.data.get(max_signals)` to something like `rule.contents.get(data).get(max_signals)`